tools/place_name_resolver/resolver.go

```diff
@@ -236,6 +238,7 @@ func geocodePlaces(mapCli MapsClient, placeId2Dcid *map[string]string, tinfo *ta
 			go geocodeOneRow(j, placeId2Dcid, tinfo, mapCli, &wg)
 		}
 		wg.Wait()
+		time.Sleep(1 * time.Second)
```

Comment:
curious why is this needed? (add a comment)

Reply:
Added a comment.